integration-cli/docker_experimental_network_test.go

```diff
@@ -0,0 +1,589 @@
+// +build experimental
+
+package main
+
+import (
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/docker/docker/pkg/integration/checker"
+	"github.com/docker/docker/pkg/parsers/kernel"
+	"github.com/go-check/check"
+)
+
+var (
+	MacvlanKernelSupport = testRequirement{
```

Comment:
that's correct. though our minimal supported version is 3.10, we don't fail the engine or the tests if it is run on earlier versions of the kernel. Hence, we don't want to fail the IT for those who are running in earlier versions (since we don't intentionally break build or IT). Keeping this requirement helps in skipping these tests in such kernels.